ddtrace/contrib/jinja2/patch.py

```diff
@@ -0,0 +1,131 @@
+import jinja2
+from wrapt import wrap_function_wrapper as _w
+
+from ddtrace import config
+
+from ...ext import http
+from ...utils.formats import asbool, get_env
+from ...pin import Pin
+from ...utils.wrappers import unwrap as _u
+from .constants import DEFAULT_SERVICE, DEFAULT_TEMPLATE_NAME
+
+
+# default settings
+config._add('jinja2',{
+    'service_name': get_env('jinja2', 'service_name', DEFAULT_SERVICE),
```

Comment:
See my comment below, I think we can default the service to `None`.

Reply:
ok, it makes sense !